cmd/podman/play_kube.go

```diff
@@ -40,7 +42,7 @@ func init() {
 	flags.BoolVarP(&playKubeCommand.Quiet, "quiet", "q", false, "Suppress output information when pulling images")
 	// Disabled flags for the remote client
 	if !remote {
-		flags.StringVar(&playKubeCommand.Authfile, "authfile", shared.GetAuthFile(""), "Path of the authentication file. Use REGISTRY_AUTH_FILE environment variable to override")
```

Comment:
is shared.GetAuthFile being used elsewhere?  If not, did you remove it?  Might answer this question the further I get down in the review here.

Reply:
shared.GetAuthFile("") has been deleted

Reply:
I don't see the removal of that function form cmd/podman/shared/funcs.go in this PR....